### ISSUE TITLE
Outdated comment?

### ISSUE THREAD

**Issue by peterlevy** [NERC-CEH/ukem#7]
https://github.com/NERC-CEH/ukem/blob/7ba7f53f218b7e684b32d59bc8623fb04437171a/R/ukem.R#L73

**Comment by peterlevy** [comment]
Mainly just looking at this as a way of adding queries to the code, without adding comment lines which go out of date (problem gets fixed, comment line doesn't get deleted).

**Comment by oSamTo** [comment]
ok good idea. I have deleted the comment in flagged above, but I expect there could be more like this (unless you've done a sweep?), so i wont close for now.

**Comment by peterlevy** [comment]
Yes, I think it needs a once-over to remove redundant comments - some yours, some mine.
